.azurepipelines/Windows-VS.yml

```diff
@@ -18,14 +30,14 @@ schedules:
     - release/*
   always: true          # Always build, even if no changes
 
-pr:
-- dev/*
-- release/*
-# MU_CHANGE [END]
-
 jobs:
-- template: templates/pr-gate-build-job.yml
+- template: Jobs/PrGate.yml@mu_devops
   parameters:
-    tool_chain_tag: 'VS2022'
-    vm_image: 'windows-2022'
-    arch_list: "IA32,X64"
+    arch_list: IA32,X64
```

Comment:
Just a question: arch_list under windows is different than ubuntu. Is this intentional?

Reply:
Yes, it is intentional in this change as it was different in the original files. This is something I assumed was a mistake, but I don't want to change behavior in this transition. I'm happy to add a follow-on change to make them consistent.